.github/workflows/check_urls.yml

```diff
@@ -0,0 +1,46 @@
+name: Check URLs
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: urls-checker-code
+      uses: urlstechie/urlchecker-action@master
+      with:
+        subfolder: onnx
+        file_types: .md,.py,.rst,.ipynb,.cc,.h,.cpp
+        print_all: false
+        timeout: 2
+        retry_count# : 2
+        # exclude_urls: https://...,...
+        # exclude_patterns: https://...
+        force_pass: true  # make the action pass until we fix urls
```

Comment:
Trying to understand this: so it will fail after urls are fixed?

Reply:
It does not fail if `force_pass: true`, with false, it fails if the urls is not found. I was hoping the find a trigger which checks the urls regularly but not at every PR.

Reply:
I see! I misread. Thanks.

Reply:
@jcwchen fyi